VSP/D8_paper.des

```diff
@@ -0,0 +1,44 @@
+43
+OOWZWGGROQZ:Duct:Design:Chord: 1.20623720669
+TTRJCLVSWWP:Duct:Design:Height: 1.43827033599
+YUWFYBTYKTL:Duct:Design:ThickChord: 0.1625
+TVQVWMMVRYB:Duct:Design:Width: 1.43827033599
+EGCVYPSLWEZ:Duct:XForm:X_Rel_Location: 29.2619489442
+RJLYSBJAFOT:Duct:XForm:Y_Rel_Location: 1.30525976916
+HKVDGHIEXRW:Duct:XForm:Y_Rotation: 15.0
+GBGVQARDEVD:Duct:XForm:Z_Rel_Location: 1.61893065875
+MCVUEHMJBGG:Floor:Design:Height: 0.130048000005
+EVDJZIXRYSR:Floor:Design:Length: 21.5860415924
+SKXVOFXEYEZ:Floor:Design:Width: 5.22103907664
+KNNNINRHVVJ:Floor:XForm:X_Rel_Location: 7.15868045774
+AFIOFOUHMWM:Floor:XForm:Z_Rel_Location: -0.565024000002
+JMWPVNGZBYQ:Fuselage:XSecCurve_1:RoundedRect_Height: 3.77396674269
+KFWNCSRQOCQ:Fuselage:XSecCurve_1:RoundedRect_Width: 5.22103907664
+HOVDTKFGFQC:Fuselage:Design:Length: 32.6659318044
+WKRLDITVGSF:Fuselage:XSecCurve_3:RoundedRect_Height: 3.77396674269
+TBCZTWFMJDM:Fuselage:XSecCurve_3:RoundedRect_Width: 5.22103907664
+OVEJIBRDSBJ:Fuselage:XSec_3:XLocPercent: 0.828514556413
+KBKZBHMUHEP:Fuselage:XSec_1:XLocPercent: 0.270593842312
+JOBWSWPMZIB:Fuselage:XSecCurve_2:RoundedRect_Height: 3.77396674269
+HPKOTUWYSIY:Fuselage:XSecCurve_2:RoundedRect_Width: 5.22103907664
+LQXJHZEHDRX:HorizontalTail:XSec_1:Root_Chord: 3.42671271919
+CHYQUCYJMPS:HorizontalTail:XSec_1:Span: 6.36314138574
+IFZAMYYJPRP:HorizontalTail:XSec_1:Sweep: 8.00000000028
+AYFSAELIRAY:HorizontalTail:XSec_1:Tip_Chord: 1.02801381579
+USGQFZQKJWC:HorizontalTail:XForm:X_Rel_Location: 30.9525603628
+BLMHVDOLAQJ:HorizontalTail:XForm:Z_Rel_Location: 4.15122080266
+LLYTEYDPDID:VerticalTail:XForm:X_Rel_Location: 30.1126101117
+BFZDOVCXTAV:VerticalTail:XForm:Y_Rel_Location: 2.61051953832
+FQDVQTUBLUX:VerticalTail:XForm:Z_Rel_Location: 0.5
+GWTZZGTPXQU:VerticalTail:XSec_1:Dihedral: -10
+MBZGSEIYFGW:VerticalTail:XSec_1:Root_Chord: 2.55330146994
+JXFRWSLYWDH:VerticalTail:XSec_1:Span: 3.65122080266
+XLPAIOGKILI:VerticalTail:XSec_1:Sweep: 25.0000000003
+CUIMIUZJQMS:VerticalTail:XSec_1:Tip_Chord: 0.765990440985
+NNIHPEXRTCP:Wing:XSec_1:Root_Chord: 5.75230134324
+AYJHHOVUHBI:Wing:XSec_1:Span: 21.3359991187
+AGOKGLSLBTO:Wing:XSec_1:Sweep: 13.2370000006
+HGZBRNOPIRD:Wing:XSec_1:Tip_Chord: 0.862845213475
+UOBOGEWYYZZ:Wing:XForm:X_Rel_Location: 19.481966639
+SMCAVCZXJSG:Wing:XForm:X_Rotation: 7.0
+MOGKYBMVMPD:Wing:XForm:Z_Rel_Location: -1.68698337134
```

Comment:
@1ozturkbe use these .des files to generate the vsp. I can't get it to work, the vsp script just crashes vsp on my machine. Once you have updated the VSP I'll update the paper results (assuming vsp looks good, really don't want to update paper results twice)